tests/testthat/test-object-package.R

```diff
@@ -0,0 +1,34 @@
+context("Object: Package")
+
+test_that("author with one given name ", {
```

Comment:
This seems like a special case of the next test, so I probably would drop it.

Reply:
Removed